### PR TITLE
run wallet tests towards the stack

### PR DESCRIPTION
**Pull request by InoMurko**

### CODE REVIEW

**Comment by CAPtheorem** [comment]
@InoMurko - looks good to me.

**Comment by InoMurko** [comment]
not ready yet pls don’t merge

File: ops/docker-compose-omgx-services.yml [hunk 1]
```diff
@@ -30,3 +30,43 @@ services:
...
+      - bl-wl
+    image: omgx/message-relayer-fast
+    build:
+      context: ..
+      dockerfile: ./ops/docker/Dockerfile.message-relayer-fast
+    environment:
+        L1_NODE_WEB3_URL: http://l1_chain:8545
+        L2_NODE_WEB3_URL: http://l2geth:8545
+        L1_WALLET_KEY: "0xde9be858da4a475276426320d5e9262ecfc3ba460bfac56360bfa6c4c28b4ee0"
+        L1_TARGET: 0x0
```
Comment (cby3149) [inline ops/docker-compose-omgx-services.yml:65]: `L1_TARGET` is replaced by `WHITELIST_ENDPOINT`. Please remove it.

File: packages/omgx/message-relayer-fast/README.md [hunk 2]
```diff
@@ -31,7 +31,6 @@ Use .env.example for quick tests
 ADDRESS_MANAGER_ADDRESS= <address manager contract address>
 L1_NODE_WEB3_URL= <l1 node url>
 L2_NODE_WEB3_URL= <l2 node url>
-L1_WALLET_KEY= <l1 wallet key>
 L1_MESSENGER_ADDRESS= <l1 custom messenger address>
 L1_WALLET_KEY= <private_key account for relayer>
 L1_TARGET= <target contract to allow relays to, set to 0x0 to skip>
```
Comment (cby3149) [inline packages/omgx/message-relayer-fast/README.md:36]: Please remove `L1_TARGET`.

**Review by cby3149 — APPROVED** [review]

File: ops/docker-compose-omgx-services.yml (hunk 1 above)
Comment (InoMurko) [reply ops/docker-compose-omgx-services.yml:65]: Cool!

File: ops/docker-compose-omgx.yml [hunk 3]
```diff
@@ -126,7 +126,7 @@ services:
       L2_NODE_WEB3_URL: http://l2geth:8545
       URL: http://deployer:8081/addresses.json
       SEQUENCER_PRIVATE_KEY: "0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d"
-      PROPOSER_PRIVATE_KEY: "0x5de4111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab365a"
```
Comment (CAPtheorem) [inline ops/docker-compose-omgx.yml]: FYI - I think deleting the proposer key has an unexpected side effect, which is that the proposer then just uses the `SEQUENCER_PRIVATE_KEY`, which can lead to the nonce errors

**Comment by CAPtheorem** [comment]
The overall system is very well done - including all the wallet tests and the GH and the local whitelister etc. Beautiful!

**Review by souradeep-das — COMMENTED** [review]
looks great! 🚀 
any idea why the LP test fail? they seem to be configured correctly

**Comment by souradeep-das** [comment]
further, opened two issues for tracking-
a. verifying and rem if there's any unused/test contacts from deployments 
https://github.com/omgnetwork/optimism/issues/113
b. heads up, the integration tests as of now are expecting a fresh contract state, so running them second time on existing contracts might fail. will add some unit tests for the wallet, though will need l2geth, will try mocking relayers
https://github.com/omgnetwork/optimism/issues/114

**Review by souradeep-das — APPROVED** [review]
🚀

File: ops/docker-compose-omgx.yml (hunk 3 above)
Comment (InoMurko) [reply ops/docker-compose-omgx.yml]: could be! but this is not my doing... this file is generated from the original `ops/docker-compose.yml`

File: packages/omgx/wallet/test/shared/env.ts [hunk 4]
```diff
@@ -158,6 +158,13 @@ export class OptimismEnv {
      await this.watcher.getL2TransactionReceipt(xDomainMsgHash)
    }
 
+   async waitForXDomainTransactionMessengerFast(
```
[the remark this answers is not in the document]
Comment (InoMurko) [reply packages/omgx/wallet/test/shared/env.ts]: sorted!

**Pull request merged**